lib/notification-issue.coffee

```diff
@@ -190,8 +190,15 @@ class NotificationIssue
           return packName unless /^\.\./.test(relativePath)
       @getPackageNameFromFilePath(filePath)
 
-    if options.detail? and packageName = getPackageName(options.detail)
-      return packageName
+    if options.detail?
+      file = /(\((.+?):\d+|\((.+)\)|(.+))/.exec(options.detail)[1]
```

Comment:
It just seems like there is filtering being done before it hits the `getPackageName` method. Should that filtering logic go in the `getPackageName` method?